spec/system/schools/school_targets_spec.rb

```diff
@@ -201,6 +204,49 @@
 
         it { expect(page).not_to have_link("View results") }
 
+        context 'and there are progress reports for both targets' do
+          let!(:expired_electricity_progress) { build(:fuel_progress, fuel_type: :electricity, progress: 0.99, target: 20, usage: 15) }
+          let!(:expired_gas_progress)         { build(:fuel_progress, fuel_type: :gas, progress: 0.59, target: 19, usage: 17) }
+          let!(:expired_last_generated)       { Date.yesterday }
+
+          let(:expired_target) do
+            create(:school_target, school: test_school,
+              start_date: Date.yesterday.prev_year, target_date: Date.yesterday,
+              electricity: 2, gas: 4,
+              electricity_progress: expired_electricity_progress, gas_progress: expired_gas_progress,
+              report_last_generated: expired_last_generated)
+          end
+
+          let!(:electricity_progress) { build(:fuel_progress, fuel_type: :electricity, progress: 0.90, target: 15, usage: 15) }
+          let!(:gas_progress)         { build(:fuel_progress, fuel_type: :gas, progress: 0.50, target: 7, usage: 17) }
+          let!(:last_generated)       { Time.zone.today }
+
+          let!(:target) do
+              create(:school_target, school: test_school,
```

Comment:
I've switched most of the `let!` to `let`. That wasn't needed for the other objects.  The target needs to be created before the visit to page, so that should be a `let!`.

I've manually re-run rubocop on the file and its not giving me any warnings.